### PR TITLE
Dockerfile: now sync’d from behance/docker-base

### PR DESCRIPTION
**Pull request by bryanlatten**
- docker-base is at 16.04 already
- Installation of S6 + goss is now simply leveraged through the common base.

### CODE REVIEW

**Comment by aivtsan** [comment]
👍

File: container/root/goss.nginx.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+gossfile:
+  goss.base.yaml: {}
```
Comment (mannytoledo) [inline container/root/goss.nginx.yaml]: I like that we can rerun the parent container tests again in downstream containers. 🔨

**Comment by mannytoledo** [comment]
👍

File: Dockerfile [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
 
 # Ensure base system is up to date
 RUN apt-get update && \
     apt-get upgrade -yqq && \
     # Install pre-reqs \
     apt-get install -yqq \
         software-properties-common \
-        curl \
     && \
-    # Add goss for local testing
```
Comment (bryanlatten) [inline Dockerfile]: included from the parent now :)

File: Dockerfile [hunk 3]
```diff
@@ -1,30 +1,22 @@
...
-# Used with alternative CMD (worker.sh), leverages supervisor to maintain long-running processes
-ENV SIGNAL_BUILD_STOP=99 \
-    CONTAINER_ROLE=web \
+ENV CONTAINER_ROLE=web \
     CONTAINER_PORT=8080 \
     CONF_NGINX_SITE="/etc/nginx/sites-available/default" \
     CONF_NGINX_SERVER="/etc/nginx/nginx.conf" \
-    NOT_ROOT_USER=www-data \
-    S6_BEHAVIOUR_IF_STAGE2_FAILS=2 \
-    S6_KILL_FINISH_MAXTIME=5000 \
```
Comment (bryanlatten) [inline Dockerfile]: env variables are defined in the parent now

**Comment by ko-be** [comment]
👍

**Comment by bossjones** [comment]
👍

**Pull request merged**